azurerm/config.go

```diff
@@ -60,10 +60,20 @@ import (
 	"github.com/Azure/go-autorest/autorest/adal"
 	"github.com/Azure/go-autorest/autorest/azure"
 	"github.com/hashicorp/terraform/terraform"
+	uuid "github.com/hashicorp/go-uuid"
 	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/authentication"
 	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
 )
 
+var MSClientRequestId = ""
```

Comment:
no need to export this, would call it `msClientRequestID` (also note capital ID, per Go [initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) convention)

Reply:
Made the change.